src/ngraph/runtime/intelgpu/intelgpu_backend.cpp

```diff
@@ -119,9 +119,13 @@ static void do_logical_operation(cldnn::topology& topology,
 
     const string& inputA_name = op->get_inputs().at(0).get_tensor().get_name();
```

Comment:
Our naming convention is for snake_case for local variables. `inputA_name` does not conform. Perhaps we can call these `input0_name` and `input1_name`.  We can do this in a separate PR. There is plenty of refactoring to do later...